path.red

```diff
@@ -43,7 +88,43 @@ let trans
   : Path A (p 0) (q 1)
   =
   λ i →
+    trans/filler A x p q 1 i
+
+let trans/unit/r
+  (A : type)
+  (p : Line A)
+  : Path (Path A (p 0) (p 1)) (λ i → p i) (trans A (p 1) p (λ _ → p 1))
+  =
+  λ i j →
+    trans/filler A (p 1) p (λ _ → p 1) i j
+
+; This proof gets simpler when dead tubes are deleted!
+let trans/sym/r
+  (A : type)
+  (p : Line A)
+  : Path (Path A (p 0) (p 0)) (λ _ → p 0) (trans A (p 0) p (symm A p))
+  =
+  λ k i →
     comp 0 1 (p i) with
     | i=0 ⇒ λ _ → p 0
-    | i=1 ⇒ λ i → q i
+    | i=1 ⇒ λ j → symm A p j
+    | k=0 ⇒ λ j → symm/filler A p i j
+    ;| k=1 ⇒ λ j → trans/filler A (p 0) p (symm A p) j i
     end
+
+; Define LineD and PathD?
+; Perhaps we could parallelize this proof? ;)
+let symmd
+  (A : Line^1 type)
+  (p : [i] A i with end)
+  : [i] (symm^1 type A) i with
+    | i=0 ⇒ p 1
+    | i=1 ⇒ p 0
+    end
+  =
+  λ i →
+    ?todo
```

Comment:
I'm having difficulties here, if anyone wants to take a look.

Reply:
Apparently, this is fixed in my branch. I'm going to merge your code into my branch and then it will be merged together.